.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -10,28 +10,17 @@ labels: Bug
 
 ### Steps to Reproduce
 
+<!-- Describe the specific steps on how to reproduce the issue. -->
+
 1. Go to '...'
 2. Click on '....'
 3. Scroll down to '....'
 4. See error
 
-### Expected Behavior
-
-<!-- A clear and concise description of what you expected to happen. -->
+### Related Issues
 
-### Actual Behavior
-
-<!-- A clear and concise description of what actually happened. -->
```

Comment:
For me this felt really repetitive, I think I never actually filled those fields with meaningful information... Other opinions?

Reply:
While I totally agree that the "Actual Behavior" section is already covered by the bug description itself, I usually fill the "Expected Behavior" part. Sometimes I also try to mention alternatives here. Maybe you have made different experiences regarding this section.
I'm totally fine to remove the section and check in a few months, whether we are missing some information and which sections get actually filled.

Reply:
Good point, Then lets keep the `Expected Behavior` section and check in a few months if we think we actually need this.

Reply:
will you readd this?

Reply:
yes